tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -686,7 +686,31 @@ pub(crate) mod tests;
 /// be composed together with other subscribers to build a [collector]. See the
 /// [module-level documentation](crate::subscribe) for details.
 ///
+/// # Enabling interest
+///
+/// Whenever an tracing event (or span) is emitted, it goes through a number of
```

Comment:
might be nice if "event" and "span" linked to the `tracing` docs, but it's not a big deal --- people reading this documentation _probably_ know what those are by now.